MapboxCoreNavigation/RouteController.swift

```diff
@@ -843,6 +881,15 @@ extension RouteController {
 
         return event.id.uuidString
     }
+    
+    func enqueueFoundFasterRouteEvent() -> String {
+        let eventDictionary = events.navigationFoundFasterRoute(routeController: self)
+        let event = FeedbackEvent(timestamp: Date(), eventDictionary: eventDictionary)
```

Comment:
Rather than a `FeedbackEvent`, we should either reuse or subclass `RerouteEvent`.